libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -27,8 +27,21 @@
 #import <SalesforceSDKCore/SalesforceSDKCore.h>
 #import "SFRestAPI+Internal.h"
 #import "SFNativeRestRequestListener.h"
-
-@interface SalesforceRestAPITests ()
+ 
+ // Constants only used in the tests below
+#define ENTITY_PREFIX_NAME @"RestClientTestsiOS"
```

Comment:
Started using AppCode, so moved to constants a bunch of commonly used strings in this test file